worker-store-http/src/main/java/com/hpe/caf/worker/datastore/http/HttpDataStore.java

```diff
@@ -74,7 +74,13 @@ public void shutdown()
     }
 
     @Override
-    public HealthResult healthCheck()
+    public HealthResult checkAlive()
+    {
+        return new HealthResult(HealthStatus.HEALTHY);
+    }
```

Comment:
As discussed we've decided to make this a default implementation at the `caf-common` level so this shouldn't be needed here.